contributing/code_of_conduct/index.rst

```diff
@@ -0,0 +1,92 @@
+Code of Conduct
+===============
+
+Our Pledge
+----------
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+education, socio-economic status, nationality, personal appearance, race,
+religion, or sexual identity and orientation.
+
+Our Standards
+-------------
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+  advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
```

Comment:
I'm still 👎 about this phrase:

> * Other conduct which could reasonably be considered inappropriate in a professional setting.

The reason, as explained multiple times in the other pull request, is that this clause is a backdoor to potential power abuse by the enforcement team.

The other items of the list are perfectly reasonable and perfectly defined: sexualized language, trolling, harassment, doxxing, etc. But the last phrase is a "catch-all" clause that says: we'll ban anything that we consider inappropriate.

Reply:
Noted .. just for the record by previously stated response to this objection:
we cannot realistically list every objectionable behavior. this is not a legal document as we lack the infrastructure to handle rules at this level of detail. I understand the concern for a "power grab", but since the enforcement team is appointed by the core-team, there is still a way to stop the enforcement team if they ever do get out of hand.

Reply:
I have thought about this some more and consulted an expert in code of conduct enforcement. My conclusion is that opening up the records to the entire core team is too much in conflict with insuring privacy. As such I would say its sufficient that if the core team suspects that there is overreach happening, that they simply use the power to change the members in the enforcement team. Of course members of the core team can then also become members of the enforcement team as part of this process. But the goal is to better protect privacy by:
* limiting access to the records to as few people as possible
* limiting access to people that have invested time into educating themselves on the topic (and ideally received some formal training).

Reply:
> I would say its sufficient that if the core team suspects that there is overreach happening, that they simply use the power to change the members in the enforcement team

And how can we know or suspect that if we don't have access to any information? If the entire Core Team is too much, then please give access to a small representation of the Core Team (three people for example).

Reply:
If people from the core team are interested in the topic enough, then why don't they just become part of the enforcement team?

Reply:
It's not about being part of the ET (Enforcement Team) ... we need an outside control of ET to ensure that they don't exceed their competences, that they don't act arbitrarily, etc. It'd be nice if someone (or a small group of people) from the CT (Core Team) watches the ET.

I propose myself as a candidate for that role: overseeing the ET. My proposed workflow would be that the designated person would have full access to all reports and all decisions from the ET. That person won't attend meetings of ET, won't make any decision, won't revoke any decision and won't change anything. If that person thinks that the ET is acting wrongly in some case, he/she will report it to the CT so the entire core team members can make a decision about that.

Reply:
imho this creates more problems than it *potentially* solves .. not even the ET will have automatic access to all reports since depending on the people involved people will recuse themselves or reports may wish to not include specific people due to certain experiences.

now we would have to also keep some people in the loop that are not even part of the process.

Reply:
Power Abuse is also a point we can add to the code of conduct.